gusto/__init__.py

```diff
@@ -21,3 +21,4 @@
 from gusto.timeloop import *                         # noqa
 from gusto.transport_methods import *                # noqa
 from gusto.wrappers import *                         # noqa
+from gusto.numerical_integrator import *             # noqa 
```

Comment:
These imports are ordered alphabetically so could you put it in its correct place?